<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralFilteringSettingEditWidget</class>
 <widget class="QWidget" name="GeneralFilteringSettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="samplingGroupBox">
     <property name="title">
      <string>Sampling</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="samplingAllRadioButton">
        <property name="text">
         <string>All vertices</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="samplingRateRadioButton">
        <property name="text">
         <string>Sampling rate:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="samplingRateSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="samplingNumberRadioButton">
        <property name="text">
         <string>Specify number of vertices</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="samplingNumberSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>samplingRateRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>samplingRateSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>samplingNumberRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>samplingNumberSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
